### DISCUSSION TITLE
Contemplate different languages

### DISCUSSION THREAD

**Discussion by ikerroba** [microcks/microcks#636] [Q&A]
Hello, is it possible to generate different individual documentations depending on the language and then select which one to read?
![tempsnip](https://user-images.githubusercontent.com/110023610/180967839-9534dc52-5893-4f20-bce7-b84af77d6953.png)
Thanks

**Comment by lbroudoux** [comment]
Hello, the documentation configuration associated to each API type is not configurable. We're using ReDoc for REST (OpenAPI/Swagger) contracts and AsyncAPIDoc for AsyncAPI.
